### ISSUE TITLE
Determine confirmed address balance

### ISSUE THREAD

**Issue by Vimiso** [Blockchair/Blockchair.Support#185]
Take the following request:

https://api.blockchair.com/bitcoin/dashboards/addresses/1NDa55vnwAVbZkeEPnWGwJdskyBZHsE5eE,13YELwFJEGf9DXV6juqmLYYayEAuwfgkQ7

How can I determine whether the balances are confirmed or not? Is it possible to send a request parameter showing balances with a min of x confirmations?

If not possible, could this be a feature we see soon?

Thanks!

**Comment by Har01d** [comment]
Yes, I think displaying the confirmed balance would be a nice feature to have, we’ll give it some priority.

**Comment by ricktian1226** [comment]
hi, how is this feature going?
we need it urgently too. Our project needs to get the balance with 6 confirmations.
And I wonder if the balance includes unconfirmed ones by API :  https://api.blockchair.com/bitcoin/dashboards/addresses/{A}

thanks a lot.

**Comment by Har01d** [comment]
@ricktian1226, could you please drop us a line at <info@blockchair.com> about your project and your needs mentioning this requests? Then we’ll see if we can speed up the process. Just as a reminder, beginning July 19th applications using our API should apply for a key: https://github.com/Blockchair/Blockchair.Support/blob/master/API.md
